dockers/docker-syncd-mlnx-rpc/Dockerfile

```diff
@@ -44,5 +45,4 @@ COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
 ENTRYPOINT service rsyslog start \
     && service syncd start       \
-    && while [ "x$( ifconfig Ethernet12 | grep RUNNING )" = 'x' ] ; do sleep 1 ; done \
```

Comment:
@lguohan This is not in this pull request. It is a commit aed2ab5b7af738fdea5df804ced10510dba38e08 pushed by @pavel-shirshov.